src/v2/Apps/Artwork/Components/OtherWorks/ArtistSeriesArtworkRail.tsx

```diff
@@ -0,0 +1,99 @@
+import { Box, Flex, Sans, Separator } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtistSeriesArtworkRail_artwork } from "v2/__generated__/ArtistSeriesArtworkRail_artwork.graphql"
+import { Carousel } from "v2/Components/Carousel"
+import FillwidthItem from "v2/Components/Artwork/FillwidthItem"
+import { StyledLink } from "v2/Apps/Artist/Components/StyledLink"
+import { getENV } from "v2/Utils/getENV"
+
+interface Props {
+  artwork: ArtistSeriesArtworkRail_artwork
+}
+
+export const ArtistSeriesArtworkRail: React.FC<Props> = ({
+  artwork: { artistSeriesConnection },
+}) => {
+  if (!artistSeriesConnection.edges.length) {
+    return null
+  }
+
+  const artistSeries = artistSeriesConnection.edges[0].node
```

Comment:
If there are multiple artist series returned, we just want the first one. I think it's an edge case for an artwork to be in multiple series (although that's the data model), plus there's the `default_order` being applied on this in the Gravity resolver.

Reply:
(We specify `first: 1` for the `artistSeriesConnection` as well).